cmd/bundle/run.go

```diff
@@ -55,17 +61,30 @@ task or a Python wheel task, the second example applies.
 			return diags.Error()
 		}
 
-		diags = bundle.Apply(ctx, b, bundle.Seq(
-			phases.Initialize(),
-			terraform.Interpolate(),
-			terraform.Write(),
-			terraform.StatePull(),
-			terraform.Load(terraform.ErrorOnEmptyState),
```

Comment:
`terraform.Load` is needed to populate resource ids (+ error on empty state), but seems like this change removed it completely?

Reply:
You're right we should keep something like this for the error state.